### PR TITLE
Refactor server URLs to include common path roots

### PR DESCRIPTION
**Pull request by lornajane**
# Description

Tidy up the servers arrays to hold as much base path as makes sense.

# Fixes

* Without changing functionality, change ho we describe base URLs and path items.

# Checklist

- [x] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/redact.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+  version: 1.0.4
   title: "Redact API"
   description: Nexmo provides [Redact API](https://developer.nexmo.com/redact/overview) to help organisations meet their privacy compliance obligations. Redact API allows you to redact private information on demand, allowing you to meet your own compliance needs.
   contact:
     name: Nexmo DevRel
     email: devrel@nexmo.com
     url: "https://developer.nexmo.com/"
 servers:
-  - url: https://api.nexmo.com
+  - url: https://api.nexmo.com/v1/redact/transaction
```
Comment (mheap) [inline definitions/redact.yml]: I'm not sure that I'd class this as the base URL. `/v1/redact` is the base and `/transaction` is the operation imo

File: definitions/audit.yml [hunk 2]
```diff
@@ -17,10 +17,10 @@ info:
     url: 'https://opensource.org/licenses/MIT'
 
 servers:
-  - url: https://api.nexmo.com
+  - url: https://api.nexmo.com/beta/audit/events
```
Comment (mheap) [inline definitions/audit.yml]: I'd expect the base URL to be `/beta/audit` and `/events` to be the operation

**Comment by lornajane** [comment]
Changes made!  I absolutely did dither over what made the most sense there.

**Comment by lornajane** [comment]
Note that the automated preview deploy failed, I'm not sure if I did something wrong though ...

**Comment by mheap** [comment]
This looks good to me now, but is going to break some tooling (multiple version changes detected). Let me work on fixing that today before we merge

**Comment by lornajane** [comment]
I'm really conscious that with linear numbering, WIP is really, really unhelpful on this repo! Please let me know if it would be better to get these changes in a one-PR-per-spec format (or one commit per spec? Does that help?) as a workaround until there's time to look at the tools

**Comment by mheap** [comment]
@lornajane I started updating the tooling to support multiple changes in a single PR, but then realised that it probably doesn't make sense as the PR description would be used for multiple releases.

Could you update one-per-PR and I'll merge as they come in?

**Comment by mheap** [comment]
Closing in favour of individual PRs

**Pull request closed without merging**